### PR TITLE
Improving search

### PR DESCRIPTION
**Pull request by eyal-sasson**

### CODE REVIEW

File: css/terminal7.css [hunk 1]
```diff
@@ -319,6 +319,9 @@ button.rename-close {
     font-size: 16pt;
     flex: 10 1 auto;
     line-height: 24pt;
+    color: #01fafa;
+    background-color: #3e3446;
```
Comment (daonb) [inline css/terminal7.css:323]: better to use vars for color values. check the top of the file for an example.

File: src/pane.js [hunk 2]
```diff
@@ -303,9 +303,9 @@ export class Pane extends Cell {
     }
     toggleSearch(searchDown) {
         const se = this.gate.e.querySelector(".search-box")
-        if (se.classList.contains("hidden"))
+        if (se.classList.contains("hidden")) {
             this.showSearch()
-        else {
+        } else {
```
Comment (daonb) [inline src/pane.js:308]: no need to add `{` in 306 and `}` here

File: src/pane.js [hunk 3]
```diff
@@ -479,16 +471,26 @@ export class Pane extends Cell {
...
+            this.cmAtEnd = null
+            this.t.setOption("selectionStyle", "plain")
+            this.searchTerm = searchTerm
+        }
+
+        if (this.searchTerm != undefined) {
+            if (!this.searchAddon.findPrevious(this.searchTerm, SEARCH_OPTS))
+                this.gate.notify(`Couldn't find "${this.searchTerm}"`)
+            else { 
+
```
Comment (daonb) [inline src/pane.js:491]: blank line

File: src/pane.js [hunk 4]
```diff
@@ -479,16 +471,26 @@ export class Pane extends Cell {
...
+                this.enterCopyMode(true)
+            }
```
Comment (daonb) [inline src/pane.js:493]: no need for `{}`

**Review by daonb — CHANGES_REQUESTED** [review]
Just a bit of cosmetics

File: css/terminal7.css (hunk 1 above)
Comment (daonb) [reply css/terminal7.css:323]: I've played with the color and I think it's best to go with what we have in the site - #11051E

**Comment by daonb** [comment]
I really like the look of the search box and would love to merge this.

Anything I can do to help?

**Comment by eyal-sasson** [comment]
> I really like the look of the search box and would love to merge this.
> 
> Anything I can do to help?

Not anything I can think of, thanks.
I'll probably finish working on this on Sunday.

File: css/terminal7.css [hunk 5]
```diff
@@ -642,6 +653,18 @@ footer pre { font-size: 14px; }
 .webexec-install div {
     position: relative;
 }
+.webexec-install ::-webkit-scrollbar-thumb {
+    background: #01fafa;
```
Comment (daonb) [inline css/terminal7.css:657]: Make it var

File: css/terminal7.css [hunk 6]
```diff
@@ -642,6 +653,18 @@ footer pre { font-size: 14px; }
...
+    border-radius: 5px;
+}
+
+.webexec-install ::-webkit-scrollbar-track {
+    background: #3e3446;
```
Comment (daonb) [inline css/terminal7.css:662]: make it a var

**Review by daonb — CHANGES_REQUESTED** [review]
Found two more color constants

**Pull request merged**